### DISCUSSION TITLE
Monai 0.9 on AWS machine ( Error while installing )

### DISCUSSION THREAD

**Discussion by Nader-Khalil** [Project-MONAI/MONAI#5014] [Q&A]
I am working on Jupyter notebook linked to AWS machine
I am trying to install the latest version of Monai 
as the screen : 
![image](https://user-images.githubusercontent.com/69905322/186968373-b1ca3570-c23e-470e-89a5-87bef211db3a.png)
any help on this ? 
thanks in advance

**Comment by JuanPabloMontoya271** [comment]
Could you provide more details about the machine you are using in aws. 
Which image are you using?
Which type of machine? (GPU enabled?)
Version of python?
Have you tried running plain 
`pip install monai`?

**Reply by Nader-Khalil** [reply]
thanks for answering : 
the os is Ubuntu  
the machine  is GPU enabled  ( mainly iam connection to AWS through ronin platform the instance = G4DN.2XLARGE )
version of python = 3.9.1 ( I already updated it )
version of pip = pip 21.3.1 (python 3.6)
i already also tried to do pip install monai BUT yes it worked but the version installed was 0.8 and sense i wanted to use AttentionUnet in monai version 0.9 i was unable to do so ..

**Reply by wyli** [reply]
thanks for reporting, more recent monai versions (0.9.*) don't support python3.6 as it has reached end-of-life

**Reply by Nader-Khalil** [reply]
but i am running on python 3.9 already , or you mean the pip version ?

**Reply by wyli** [reply]
yeah, in that case perhaps you can try `python -m pip install monai`

**Reply by Nader-Khalil** [reply]
thanks for your fast reply I tried the line  and it made me able to update to monai 0.9 
![image](https://user-images.githubusercontent.com/69905322/186979255-e6766c59-ced6-4665-8a49-48515ed99b4c.png)
BUT 
i still not able to use the AttentionUnet and have such error 
![image](https://user-images.githubusercontent.com/69905322/186979425-3ab88bd5-2beb-472f-98e3-9f98ce0a4e91.png)

**Reply by wyli** [reply]
probably this jupyter notebook instance is from python3.6, so it's still in python3.6/monai 0.8..

**Reply by Nader-Khalil** [reply]
😅🤦‍♂️ so can you give a little hint to solve it ?
